setup-scripts/README.md

```diff
@@ -88,6 +88,62 @@ At this point we have almost everything running, we just need to set up your wal
 1. `docker-compose up -d` to restart the services so they pick up new env variables
 1. `docker exec caddy caddy reload --config /etc/caddy/Caddyfile` to reload Caddyfile configuration
 
+## Subdomains
+
+It might prove useful for certain skapps to be accessible through a custom
+subdomain. So instead of being accessed through `https://portal/[skylink]`, it
+would be accessible through `https://[skylink_base32].portal`. We call this
+subdomains and it is made possible by encoding Skylinks using a base32 encoding.
+You can convert Skylinks, are see how we do the encoding, using this [
+converter
+skapp](https://siasky.net/_AIPM3F3T2MJ93q3TD2QqoKnSaXv2kMc-tQBAs-stDGa0w/) To
+configure this on your portal, you have to make sure to configure the
+following:
```

Comment:
```suggestion
would be accessible through `https://[skylink_base32].portal`. You need to use
base32 encoded skylink because subdomains are case insensitive and passing
regular skylink will not work. You can convert using this [converter skapp](https://siasky.net/_AIPM3F3T2MJ93q3TD2QqoKnSaXv2kMc-tQBAs-stDGa0w/).

To configure this on your portal, you have to make sure to configure the
following:
```